docs/data/system/migration/migrating-to-v6/migrating-to-v6.md

```diff
@@ -37,13 +37,27 @@ You can follow progress on https://github.com/mui/material-ui/issues/30671.
 
 ### Grid
 
-The `Grid` component was updated to match the API of the new `PigmentGrid` component, to allow interoperability between the two:
+The `Grid` component was stabilized and updated to match the API of the new `PigmentGrid` component, to allow interoperability between the two:
 
 - The previous size and offset props were replaced with the `size` and `offset` props
```

Comment:
I think this sentence is wrong. It also lacks the `.` at the end.

Reply:
It's correct but weirdly phrased 😅, does this make it clearer?:

```suggestion
- The previous size (`xs`, `sm`, `md`, ...)  and offset (`xsOffset`, `smOffset`, `mdOffset`, ...) props, which were named after the theme's breakpoints, were replaced with the `size` and `offset` props. This makes the API more consistent.
```

Reply:
Much better IMO!

Reply:
Updated 👌🏼